### ISSUE TITLE
Function Calling support 

### ISSUE THREAD

**Issue by floschne** [ModelTC/lightllm#552]
Hi,

I couldn't find any information in the docs about function calling. Does LightLLM support it or do I need to prompt and parse manually?

**Comment by hiworldwzj** [comment]
@floschne Yes, You need do it by yourself now. We will support this feature in future.  However, the schedule is not very certain，we are currently improving basic performance and support for constraint output.

**Comment by floschne** [comment]
Totally understandable, thanks for your answer and work in general:-)
